src/v1/products.ts

```diff
@@ -107,6 +107,15 @@ export interface ProductsResponse {
   next?: () => Promise<ProductsResponse>
 }
 
+export interface ProductsBulkResponse {
+  results?: {
+    updatedProducts?: Product[]
+    invalidProducts?: Product[]
```

Comment:
Please change to snake case per convention - results from API are always snake case.
(I would say also in the API the properties should change, but it's not my area of responsibility).
```suggestion
    updated_products?: Product[]
    invalid_products?: Product[]
```